pynxtools/dataconverter/README.md

```diff
@@ -27,8 +27,7 @@ Options:
                                   arguments instead. The path to the input
                                   data file to read. (Repeat for more than one
                                   file.)
-  --reader [apm|ellips|em_nion|em_om|em_spctrscpy|example|hall|json_map|json_yml|mpes|sts|transmission|xps|xrd]
-                                  The reader to use. default="example"
+  --reader [example]              The reader to use. default="example"
```

Comment:
Is this the new output of `dataconverter --help`?

Reply:
no its not, but has to be changed as well in the data converter.

Reply:
https://github.com/FAIRmat-NFDI/pynxtools/issues/325